base/operators.jl

```diff
@@ -871,9 +871,37 @@ julia> fs = [
 julia> ∘(fs...)(3)
 3.0
 ```
+See also [ComposedFunction](@ref).
 """
 function ∘ end
 
+"""
+    ComposedFunction{F,G} <: Function
+
+Represents the composition of two callable objects `f::F` and `g::G`.
+```jldoctest
+julia> sin ∘ cos === Base.ComposedFunction(sin, cos)
+true
+
+julia> typeof(sin∘cos)
+Base.ComposedFunction{typeof(sin), typeof(cos)}
+```
+The composed pieces are stored in the fields of `ComposedFunction` and can be retrieved as follows:
+```jldoctest
+julia> composition = sin ∘ cos
+sin ∘ cos
+
+julia> composition.f === sin
+true
+
+julia composition.g === cos
```

Comment:
This should [fix the doctests](https://build.julialang.org/#/builders/39/builds/3010):
```suggestion
julia> composition.g === cos
```